src/sciline/task_graph.py

```diff
@@ -37,7 +37,7 @@ def __init__(
     def compute(
         self,
         keys: Optional[
-            Union[type, Tuple[type, ...], Item[T], Tuple[Item[T], ...]]
+            Union[type, Tuple[type, ...], Item[T], Tuple[Item[T], ...], None]
```

Comment:
Why the `None` here?

Reply:
Why do you prefer that?

Reply:
It doesn't really matter since they are basically the same.
Just less lines... I guess...? I'll revert it anyways : D